index.html

```diff
@@ -787,6 +787,20 @@ <h2>
             <td>
 Identifies a certain version timestamp of a <a>DID document</a> to be resolved
 (i.e., the <a>DID document</a> that was valid for a <a>DID</a> at a certain time).
+Note: This parameter may not be supported by all <a>DID methods</a>.
+            </td>
+          </tr>
+          <tr>
+            <td>
+<code>initial-values</code>
+            </td>
+            <td>
+Some <a>DID methods</a> may require an interim period of time (e.g., a block 
```

Comment:
What this paragraph says, and what was explained during the WG call today feel like two very different things. We shouldn't create generic parameters where DID Methods can put whatever they want to into the value... that will lead to bad interoperability. If this is going to be a base64 encoded value of the initial DID Document, then the property should be `did-document`. If it is a hash of the initial document, use a hashlink. If it is a link to the initial document, it should be `document-link`. The property should be more specific, with a specific encoding such that all methods implement it in the same way, if they choose to implement it.